Makefile

```diff
@@ -12,130 +12,222 @@
 # language governing permissions and limitations under the License.
 #
 
-.PHONY: all build-linux clean format check-format docker docker-build lint unit-test vet download-portmap build-docker-test build-metrics docker-metrics metrics-unit-test docker-metrics-test docker-vet
-
-IMAGE   ?= amazon/amazon-k8s-cni
-VERSION ?= $(shell git describe --tags --always --dirty)
+.PHONY: all dist check clean \
+		lint format check-format vet docker-vet \
+		build-linux docker \
+		unit-test unit-test-race build-docker-test docker-func-test \
+		build-metrics docker-metrics \
+		metrics-unit-test docker-metrics-test
```

Comment:
missing `portmap` above.

Reply:
> `portmap` isn't phony, its a file that's created by downloading it - if `portmap` were to be listed here it will be downloaded every time the target is activated.

ack, yes, good point.